packages/typography/src/Heading/Heading.tsx

```diff
@@ -1,9 +1,39 @@
-function Heading() {
-  return (
-    <div>
-      <h1 className="text-red-12">My Heading</h1>
-    </div>
-  );
-}
+import clsx from "clsx";
+import React from "react";
+
+import { alignMap, colorMap, sizeMap } from "../helpers/prop-mappings";
+
+type HeadingProps = React.ComponentProps<"h1"> & {
+  align?: keyof typeof alignMap;
+  size?: keyof typeof sizeMap;
+  color?: keyof typeof colorMap;
+};
+
+type HeadingRef = HTMLHeadingElement;
+
+const Heading = React.forwardRef<HeadingRef, HeadingProps>(
+  (
+    {
+      align = "DEFAULT",
```

Comment:
Nit: can we put this const as a const we share between here and the prop mapping

Reply:
Not sure I'm following?

You want `const DEFAULT = "DEFAULT"` declared?

Reply:
yep exactly and then use in the prop mappings as `[DEFAULT]: "foo"`

Reply:
Oh yeah that's even better!